content/arduino-cloud/03.tutorials/document-projects-through-webeditor/classroom_project.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Creating Projects using the Web Editor'
+description: 'This is the template for creating an article for the Arduino Documentation website.'
```

Comment:
Please update the description here. This will be what is shown in the card when you search for the tutorial as well as in the cloud section